content/posts/GSoC_2021_PreQuarter/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "GSoC'21: Pre-Quarter Progress"
+date: 2021-07-19T07:32:05+05:30
+draft: false
+categories: ["News", "GSoC"]
+description: "Pre-Quarter Progress with Google Summer of Code 2021 project under NumFOCUS: Aitik Gupta"
+displayInList: true
+author: Aitik Gupta
+
+resources:
+- name: featuredImage
+  src: "AitikGupta_GSoC.png"
+  params:
+    showOnTop: true
+---
+
+**“<ins>Well? Did you get it working?!</ins>”**
+
+Before I answer that question, if you're missing the context, check out my previous blog's last few lines.. promise it won't take you more than 30 seconds to get the whole problem!
+
+With this short writeup, I intend to talk about _what_ we did and _why_ we did, what we did. XD
+
+## Ostrich Algorithm
+Ring any bells? Remember OS (Operating Systems)? It's one of the core CS subjects which I bunked then and regret now. (╥﹏╥)
+
+The [wikipedia page](https://en.wikipedia.org/wiki/Ostrich_algorithm) has a 2-liner explaination if you have no idea what's an Ostrich Algorithm.. but I know most of y'all won't bother clicking it XD, so here goes:
+> Ostrich algorithm is a strategy of ignoring potential problems by "sticking one's head in the sand and pretend there is no problem"
+
+An important thing to note: it is used when it is more **cost-effective** to _allow the problem to occur than to attempt its prevention_.
+
+As you might've guessed by now, we ultimately ended up with the *not-so-clean* API (more on this later).
+
+## What was the problem?
+The highest level overview of the problem was:
+
+```
+❌ library1 -> buffer -> library2_with_buffer
+✅ library1 -> buffer -> tempfile -> library2_with_file

```

Comment:
I think you can just directly write fonttools and ttconv since you explain them immediatly and they're understandable from context